### ISSUE TITLE
quiche module installation fails: path doesn´t exist

### ISSUE THREAD

**Issue by benhartwich** [angristan/nginx-autoinstall#147]
Hi,

I´m trying your script at debian 10 and the installation itself works, but the path --with-quiche=/usr/local/src/nginx/modules/quiche is not existent. This occurs on nginx cache purge and other modules also. 

Is there any difference on debian 10?

**Comment by randomshell** [comment]
> --with-quiche=/usr/local/src/nginx/modules/quiche is not existent

I am on Debian 10 and I can't reproduce.

> This occurs on nginx cache purge 

I don't think this module is supported https://github.com/FRiCKLE/ngx_cache_purge/pull/51. @angristan?

**Comment by angristan** [comment]
Works for me:

```
nginx version: nginx/1.19.1
built by gcc 9.3.0 (Ubuntu 9.3.0-10ubuntu2)
built with OpenSSL 1.1.0 (compatible; BoringSSL) (running with BoringSSL)
TLS SNI support enabled
configure arguments: --prefix=/etc/nginx --sbin-path=/usr/sbin/nginx --conf-path=/etc/nginx/nginx.conf --error-log-path=/var/log/nginx/error.log --http-log-path=/var/log/nginx/access.log --pid-path=/var/run/nginx.pid --lock-path=/var/run/nginx.lock --http-client-body-temp-path=/var/cache/nginx/client_temp --http-proxy-temp-path=/var/cache/nginx/proxy_temp --http-fastcgi-temp-path=/var/cache/nginx/fastcgi_temp --user=nginx --group=nginx --with-cc-opt=-Wno-deprecated-declarations --with-cc-opt=-Wno-ignored-qualifiers --with-openssl=/usr/local/src/nginx/modules/quiche/deps/boringssl --with-quiche=/usr/local/src/nginx/modules/quiche --with-threads --with-file-aio --with-http_ssl_module --with-http_v2_module --with-http_mp4_module --with-http_auth_request_module --with-http_slice_module --with-http_stub_status_module --with-http_realip_module --with-http_sub_module --add-module=/usr/local/src/nginx/modules/ngx_cache_purge --with-http_v3_module
```

Please post the full debug output for us to try and find the issue.

**Comment by benhartwich** [comment]
Can you please name the debug command?

**Comment by angristan** [comment]
There's none, everything is displayed in the output when you run the script 🙂 If something went wrong, we should be able to see it there

**Comment by benhartwich** [comment]
Hi,

sorry for the late reply - I found the following problems:

`
....
Cloning into 'ngx_brotli'...
remote: Enumerating objects: 17, done.
remote: Counting objects: 100% (17/17), done.
remote: Compressing objects: 100% (16/16), done.
remote: Total 17 (delta 1), reused 13 (delta 0), pack-reused 0
Unpacking objects: 100% (17/17), done.
error: pathspec 'v0.1.2' did not match any file(s) known to git
...
`

There is no further error and nginx is running:

`nginx version: nginx/1.19.3
built by gcc 8.3.0 (Debian 8.3.0-6)
built with OpenSSL 1.1.0 (compatible; BoringSSL) (running with BoringSSL)
TLS SNI support enabled
configure arguments: --prefix=/etc/nginx --sbin-path=/usr/sbin/nginx --conf-path=/etc/nginx/nginx.conf --error-log-path=/var/log/nginx/error.log --http-log-path=/var/log/nginx/access.log --pid-path=/var/run/nginx.pid --lock-path=/var/run/nginx.lock --http-client-body-temp-path=/var/cache/nginx/client_temp --http-proxy-temp-path=/var/cache/nginx/proxy_temp --http-fastcgi-temp-path=/var/cache/nginx/fastcgi_temp --user=nginx --group=nginx --with-cc-opt=-Wno-deprecated-declarations --with-cc-opt=-Wno-ignored-qualifiers --with-openssl=/usr/local/src/nginx/modules/quiche/deps/boringssl --with-quiche=/usr/local/src/nginx/modules/quiche --with-threads --with-file-aio --with-http_ssl_module --with-http_v2_module --with-http_mp4_module --with-http_auth_request_module --with-http_slice_module --with-http_stub_status_module --with-http_realip_module --with-http_sub_module --add-module=/usr/local/src/nginx/modules/ngx_brotli --add-module=/usr/local/src/nginx/modules/headers-more-nginx-module-0.33 --add-module=/usr/local/src/nginx/modules/ngx_cache_purge --add-module=/usr/local/src/nginx/modules/fancyindex --add-module=/usr/local/src/nginx/modules/nginx-module-vts --with-http_v3_module
`
But if I try to open the custom module paths, the path isn´t found - e.g.:

`ll /usr/local/src/nginx/modules/ngx_brotli
ls: cannot access '/usr/local/src/nginx/modules/ngx_brotli': No such file or directory
`

**Comment by angristan** [comment]
That's because the directory is deleted at the end:

https://github.com/angristan/nginx-autoinstall/blob/65e5facbee063f2ed149a5a046872de833722c8b/nginx-autoinstall.sh#L603

As for brotli, it was still working but I fixed the error with b92170a19ad0400ed5cfb113754533bddd9ccf5d
